### ISSUE TITLE
Something's gone wrong with the landmarks

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#3223]
Intended structure of the landmarks is to identify:
- workspace (Mirador)
  -  window (manifest)
      - Item (current image in this window)

and their respective navigation/tools.

This was correct in the version we provided to Leipzig for testing:

<img width="687" alt="landmarks before" src="https://user-images.githubusercontent.com/6945691/87979641-382b9f00-ca87-11ea-9470-cffe8968f0fd.png">

They reported issues with the order of Workspace main and Workspace nav, and with the name of Workspace nav. Those were resolved, but now the window/item structure is broken: 

- the window label is repeated for the item, and 
- the item label is missing.

<img width="717" alt="landmarks after" src="https://user-images.githubusercontent.com/6945691/87979663-41b50700-ca87-11ea-8836-c7805231cb16.png">

**Comment by mejackreed** [comment]
@jvine a question about how this works. One the duplicate window labels is for a label of the "preview" window that only is visible when a window is dragged. 

Should we:
 - even provide an aria-label for this?
 - provide a different aria-label?

<img width="901" alt="image" src="https://user-images.githubusercontent.com/1656824/88114338-dfd0cc00-cb70-11ea-9ec6-349bb2fcbee8.png">

**Comment by jvine** [comment]
The drop target shouldn't be perceivable by the screen reader, especially when no drag action is in process. (That said, I don't believe we've provided keyboard access to the drag action...). 

I'm inclined to say it should not have an `aria-label` and it should have `aria-hidden="true"`.

The missing "Item" landmark is another issue related to #3107 that I haven't completely figured out yet.

**Comment by jvine** [comment]
1. The audit request in #3107 was to add `aria-label` and `aria-describedby` elements to the canvas, with `aria-describedby`referring to the text below the navigation buttons ("# of n - canvas label"). 

This _looks_ like it was done correctly, but for some reason, neither VO nor Chromevox reads the `aria-labelledby` text. They do read "digitized view". Can't figure out why. 

2. At the same time, the previous `<section aria-label="Item: Canvas label">` structure was changed to `<section aria-labelledby="same ID as the canvas labelledby">`

Problems with this:
- switching from `aria-label` to `aria-labelledby` removes the region from the landmarks list. This is not desirable.
- as in # 1, the `aria-labelledby` text is just not being read.

Comments:
- the "Item: " prefix in the original label was intentional, but adding the `# of n` context fills the same role in a more informative way. If that can be maintained in an `aria-label` then yay.
- the addition of `aria-live="polite"` to the section is 👍
